src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -621,14 +621,12 @@ private List<Integer> getEmptyLines(DetailAST ast) {
      */
     private static List<Integer> getEmptyLinesToLog(List<Integer> emptyLines) {
         final List<Integer> emptyLinesToLog = new ArrayList<>();
-        if (emptyLines.size() >= 2) {
```

Comment:
I assume this became a surviving mutation with other kills.

This prevents lines with only 1 empty line to be excluded.

The way the code is written, 1 line will never get added to the result. `previousEmptyLineNo` is set to the value of that 1 line. Then the condition to add to the result is `previousEmptyLineNo + 1 == emptyLineNo` which is based on the same 1 line. It logically impossible to for the equality `X + 1 == X` to be true.